Src/AutoFixture.ruleset

```diff
@@ -1,236 +1,247 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<RuleSet Name="AutoFixture" Description="Rule set for all AutoFixture projects" ToolsVersion="10.0">
+<RuleSet Name="AutoFixture" Description="Rule set for all AutoFixture projects" ToolsVersion="15.0">
   <IncludeAll Action="Warning" />
   <Rules AnalyzerId="Microsoft.Analyzers.ManagedCodeAnalysis" RuleNamespace="Microsoft.Rules.Managed">
-    <Rule Id="CA1000" Action="Error" />

```

Comment:
I marked all "Error" as "Warning" in [this](https://github.com/AutoFixture/AutoFixture/pull/773/commits/d75c05d479fe11bbb8ba4d1e3ffd857e0f1b270a) commit. The motivation is that only `Verify` build should fail if there are some code warnings and all other builds should pass. During the `Verify` build we treat warnings as errors, so we will still don't allow to merge code which violates rules.

Also notice, as I mentioned [here](https://github.com/AutoFixture/AutoFixture/pull/773#issuecomment-313640629) I suppressed some part of rules which started to fail after I switched to a new Code Analysis.